packages/blue-sdk-viem/contracts/GetMarket.sol

```diff
@@ -22,7 +23,10 @@ contract GetMarket {
         res.market = morpho.market(id);
 
         if (res.marketParams.oracle != address(0)) {
-            res.price = IOracle(res.marketParams.oracle).price();
+            try IOracle(res.marketParams.oracle).price() returns (uint256 price) {
+                res.hasPrice = true;
+                res.price = price;
```

Comment:
Why not considering that `hasPrice` <=> `price` defined?

Reply:
By default, unset values onchain are assigned the zero word, which corresponds to the zero value in `uint256`
There's no undefined or null values onchain